pkg/volume/fc/disk_manager.go

```diff
@@ -117,5 +119,48 @@ func diskTearDown(manager diskManager, c fcDiskUnmounter, volPath string, mounte
 		}
 	}
 	return nil
+}
+
+// utility to map a device to symbolic link
+func deviceSetUp(manager diskManager, b fcDiskMapper, volPath string) error {
+	globalPDPath := manager.MakeGlobalPDName(*b.fcDisk)
+	if !filepath.IsAbs(globalPDPath) || !filepath.IsAbs(volPath) {
+		return fmt.Errorf("These paths should be absolute: globalPDPath: %s, volPath: %s", globalPDPath, volPath)
+	}
+	devicePath, err := os.Readlink(globalPDPath + "/" + "symlink")
+	if err != nil {
+		glog.Errorf("failed to readlink: %s", globalPDPath+"/"+"symlink")
+		return err
+	}
+	noMnt, err := b.mounter.IsLikelyNotMountPoint(volPath)
```

Comment:
Why this check? IMO it can't be ever a mount point

Reply:
@jsafrane I supposed there is a case that the volume is previously used as filesystem and remained as mounted based on unmount failure. Doesn't this need to consider?

Reply:
If it was mounted as a filesystem, the pod mount would be under the volumes directory and not the devices directory.

Reply:
That's right! will remove.